protocol-designer/src/components/modals/CreateFileWizard/EquipmentOption.tsx

```diff
@@ -75,10 +66,51 @@ export function EquipmentOption(props: EquipmentOptionProps): JSX.Element {
     showCheckbox = false,
     disabled = false,
     robotType,
+    multiples,
     ...styleProps
   } = props
-  const { t } = useTranslation('tooltip')
-  const [targetProps, tooltipProps] = useHoverTooltip()
+  const { t } = useTranslation(['tooltip', 'shared'])
+  const [equipmentTargetProps, equipmentTooltipProps] = useHoverTooltip()
+  const [tempTargetProps, tempTooltipProps] = useHoverTooltip()
+  const [numMultiples, setNum] = React.useState<number>(0)
```

Comment:
If there is no specific reason to use a different name for `setter`

```suggestion
  const [numMultiples, setNumMultiples] = React.useState<number>(0)
```

Reply:
oops ya good call out, i was definitely being lazy 😅